### PR TITLE
cider: Add version 1.5.6

### PR DESCRIPTION
**Pull request by SonaliBendre**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by issaclin32** [comment]
We only accept **stable version** in Extras bucket.

You can add this app to [Versions bucket](https://github.com/ScoopInstaller/Versions) as `cider-beta`,
and add the app to Extras when stable version is available.

**Comment by SonaliBendre** [comment]
I changed to stable version

File: bucket/cider.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\`$R0\" -Recurse -Force",
+    "shortcuts": [
+        [
+            "Cider.exe",
+            "Cider"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/ciderapp/cider-releases/releases/latest"
```
Comment (issaclin32) [inline bucket/cider.json:23]:
```suggestion
        "github": "https://github.com/ciderapp/cider-releases/"
```

File: bucket/cider.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ciderapp/cider-releases/releases/download/v1.5.6/Cider-Setup-winget-1.5.6.exe#/dl.7z",
+            "hash": "sha512:0416893ef73d6883f155d0ec1276b7ff83141adfa3bf3a074a14b5af6cf742079a01ed8afeae6a026f79cad456900d47fa68d4e659742e63cdc6b4424b2aabf6",
+            "installer": {
+                "script": "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
+            }
+        }
+    },
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\`$R0\" -Recurse -Force",
```
Comment (issaclin32) [inline bucket/cider.json:15]:
```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/ciderapp/cider-releases/releases/download/v1.5.6/Cider-Setup-winget-1.5.6.exe#/dl.7z",
            "hash": "sha512:0416893ef73d6883f155d0ec1276b7ff83141adfa3bf3a074a14b5af6cf742079a01ed8afeae6a026f79cad456900d47fa68d4e659742e63cdc6b4424b2aabf6"
        }
    },
    "pre_install": [
        "# There is a symlink 'python3' that points to '/usr/python3'",
        "# ,which is not applicable on Windows and cause 7-zip extraction to have errors",
        "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\" -Switches '-xr!python3'",
        "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\`$R0\" -Recurse -Force"
    ],
```

**Pull request merged**